src/components/Header/index.js

```diff
@@ -13,8 +13,8 @@ import NavigationTabs from "../NavigationTabs";
 
```

Comment:
Also, on line 61, can you update the copy to say `Coinbase Wallet` instead of `Cipher Browser`?

Reply:
alright I replaced all instances of cipher in the codebase with coinbase wallet